src/CMakeLists.txt

```diff
@@ -34,6 +34,34 @@ peloton_default_properties(peloton-proto)
 
 add_subdirectory(${PROJECT_SOURCE_DIR}/third_party/libpg_query/ libpg_query.a)
 
+##################################################################################
+
+# TODO: Guess this should be moved into a module?
+# --[ tensorflow
+# Add library to be linked
+find_library(TFlowC
+        NAMES tensorflow
+        PATHS "/usr/local/lib")
+set(Peloton_LINKER_LIBS "${TFlowC}" ${Peloton_LINKER_LIBS}) # note, crucial to prepend!
+
+# Generate essential model files
+set(MODEL_GEN_DIR ${CMAKE_CURRENT_LIST_DIR}/brain/modelgen)
+set(MODEL_GEN_PY ${MODEL_GEN_DIR}/model_generator.py)
+set(MODEL_GEN_SETTINGS ${MODEL_GEN_DIR}/settings.json)
+set(MODEL_GEN_OUT ${MODEL_GEN_DIR}/LSTM.pb)
```

Comment:
I don't think you want to check for a specific model file here. I think either you check for each individual pb file for each model, or just simply recreate all the pb files when calling cmake anyway.